src/core/components/hv-navigator/index.tsx

```diff
@@ -64,6 +99,33 @@ export default class HvNavigator extends PureComponent<Types.Props> {
     );
   };
 
+  /**
+   * Build the dynamic and modal screens for a stack navigator
```

Comment:
(maybe not in context with this PR): I understand that `dynamic` represents the regular stack, while `modal` represents a stack specifically configured for modals, but I find the naming a bit confusing.
Our previous naming convention on our app was `home` vs `modals`, which honestly isn't that much better, but I wonder if we should streamline this. I think `dynamic` is a misnomer since the modal stack is also dynamic, so maybe we can use `regular` and `modal`? Up for other suggestions

Reply:
Agreed naming isn't ideal. I'll put some thought into it